Resources/public/js/views/serverside/ez-contenttypeeditserversideview.js

```diff
@@ -68,5 +74,43 @@ YUI.add('ez-contenttypeeditserversideview', function (Y) {
             selectionRootInput.setAttribute('value', e.selection.location.get('locationId'));
             selectedRootName.setHTML(e.selection.contentInfo.get('name'));
         },
+
+        /**
+         * tap event handler on the ezselection add option button. It adds a
+         * new option field to the form.
+         *
+         * @method _addSelectionOption
+         * @protected
+         * @param {EventFacade} e
+         */
+        _addSelectionOption: function (e) {
+            var button = e.target,
+                rootOptionsList = this.get('container').one(button.getAttribute('data-relation-root-options-list-id')),
+                prototype = rootOptionsList.getAttribute('data-prototype'),
```

Comment:
@dpobel The concept is called `prototype` in Symfony. We can of course call it `formFieldPrototype`, or find another word altogether, but it will still refer to `form.option.vars.prototype` in twig.